include/actions/AddDriftDiffusionAction.h

```diff
@@ -23,17 +34,31 @@ class AddDriftDiffusionAction : public Action
   virtual void act();
 
 protected:
+  /// Helper function that supplies the potentials charge sources
   virtual void addChargeSourceKernels(const std::string & potential_name,
                                       const std::string & charged_particle_name);
+
+  /*
+   *  Helper function that supplies the Kernels for drift-diffusion for the electrons,
+   *  energy independent charged particles, neutral particles, and
+   *  electron mean energy depending
+   */
   virtual void addADKernels(const std::string & name,
                             const std::string & potential_name,
                             const bool & Using_offset,
                             const bool & charged,
                             const bool & energy);
+
+  /// Helper function that supplies the Aux kernels to convert scaled position units
   virtual void addPosition(const std::string & position_name, const int & component);
 
+  /// Helper function that supplies the Aux kernels to convert densities from log form
   virtual void addDensityLog(const std::string & particle_name);
+
+  /// Helper function that supplies the Aux kernels for current
   virtual void addCurrent(const std::string & particle_name, const std::string & potential_name);
+
+  /// Helper function that supplies the Aux kernels for the electric field
   virtual void addEfield(const std::string & Efield_name,
                          const std::string & potential_name,
                          const int & component);
```

Comment:
@cticenhour correct me if I am wrong but I believe the MOOSE standard should be to have a description for each of the parameters of every custom function. In the style of 

```
@param name description
```

This comment is also applicable for the rest of the actions that do not have the parameter doc strings for their internal functions.

Reply:
This has been addressed in [4b5a202](https://github.com/shannon-lab/zapdos/pull/263/commits/4b5a202d8b533fba4d69259261cd06b752ddfaf7)